lib/css_parser/parser.rb

```diff
@@ -514,4 +514,4 @@ def reset! # :nodoc:
       @css_warnings = []
     end
   end
-end
+end
```

Comment:
set your editor to always add newline at end of file, makes these things go away :)